### PR TITLE
Adding known issue for backup and restore

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/19965

### CODE REVIEW

**Review by sahare — APPROVED** [review]
lgtm

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -625,6 +625,31 @@ oc get managedclusterinfo <clusterName> -n <clusterNamespace> -o json | jq -r '.
...
+As hub clusters change from passive to primary clusters and back, different clusters can backup data at the same storage location. This can result in backup collisions, which means that the latest backups are generated by a passive hub cluster. 
+
+The passive hub cluster produces backups because the `BackupSchedule.cluster.open-cluster-management.io` resource is enabled on the hub cluster, but it should no longer write backup data since the hub cluster is no longer a primary hub cluster. Run the following command to check if there is a backup collision:
+
+----
+oc get backupschedule -A
+----
+
+You might receive the following status:
+//will the explanation be displayed to the user?
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: @sahare will this entire message appear for the user

`from cluster with id [be97a9eb-60b8-4511-805c-298e7c0898b3] is using the same storage location. This is a backup collision with current cluster [1f30bfe5-0588-441c-889e-eaf0ae55f941] backup. Review and resolve the collision then create a new BackupSchedule resource to resume backups from this cluster.`

Comment (sahare) [reply release_notes/known_issues.adoc]: @dockerymick Yes, attached images are showing how it looks on CLI or UI:

Screen Shot 2022-03-14 at 3 32 48 PM
Screen Shot 2022-03-14 at 3 33 19 PM

Comment (dockerymick) [reply release_notes/known_issues.adoc]: @sahare thanks for confirming and thanks for the screen capture!

**Comment by dockerymick** [comment]
@sahare thanks for your review and approval. I have removed my comment and ready to merge this PR

**Pull request merged**